macos/FluentUITestViewControllers/TestButtonViewController.swift

```diff
@@ -31,16 +31,33 @@ class TestButtonViewController: NSViewController {
 
 		let buttonsWithImage: () -> [NSButton] = {
 			return [
-				Button(image: NSImage(named: NSImage.addTemplateName)!, style: .primaryFilled),
-				Button(image: NSImage(named: NSImage.addTemplateName)!, style: .primaryOutline),
-				Button(image: NSImage(named: NSImage.addTemplateName)!, style: .borderless),
+				Button(image: NSImage(named: NSImage.stopProgressTemplateName)!, style: .primaryFilled),
+				Button(image: NSImage(named: NSImage.stopProgressTemplateName)!, style: .primaryOutline),
+				Button(image: NSImage(named: NSImage.stopProgressTemplateName)!, style: .borderless),
 			]
 		}
 
 		let disabledButtonsWithImage = buttonsWithImage().map { button -> NSButton in
 			button.isEnabled = false
 			return button
 		}
+		
+		let isRTL = NSApp.userInterfaceLayoutDirection == .rightToLeft
+		let leadingArrowImage = isRTL ? NSImage(named: NSImage.goRightTemplateName)! : NSImage(named: NSImage.goLeftTemplateName)!
+		let trailingArrowImage = isRTL ? NSImage(named: NSImage.goLeftTemplateName)! : NSImage(named: NSImage.goRightTemplateName)!
```

Comment:
We should replace this with the fluent arrow icon whenever we get that into our repo

Reply:
Thanks!